client/public/config/configuration.json

```diff
@@ -1,6 +1,10 @@
 [
   {
     "URL":"https://namex-dev.pathfinder.gov.bc.ca",
-    "PAYMENT_PORTAL_URL": "https://dev.bcregistry.ca/business/auth/makepayment"
+    "PAYMENT_PORTAL_URL": "https://dev.bcregistry.ca/business/auth/makepayment",
+    "BUSINESSES_URL": "https://dev.bcregistry.ca/business/auth/",
+    "KEYCLOAK_CONFIG_PATH": "/local-keycloak-config-url/keycloak.json",
+    "VUE_APP_AUTH_ROOT_API": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
+    "VUE_APP_LEGAL_ROOT_API": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1"
```

Comment:
Why name there VUE_APP_xxx ? It suggests they should be environment variables (in .env file).

Reply:
Thats a fair point, i was following the naming convention used elsewhere when using an .env. Will refactor these names for simplicity in the next commit!

Reply:
Updated in next commit